File: is_pangram/is_pangram.cs
using System;

public class is_pangram
{
	public static bool isPangram(string str)
	{
		bool[] pangram = new bool[26];

		for (int i = 0; i < str.Length; i++)
		{
			if (str[i] >= 'A' && str[i] <= 'Z')
				pangram[str[i] - 'A'] = true;
			if (str[i] >= 'a' && str[i] <= 'z')
				pangram[str[i] - 'a'] = true;
		}

		for (int i = 0; i < 26; i++)
			if (!pangram[i])
				return false;

		return true;
	}

	static void Main(string[] args)
	{
		bool pangram = isPangram("The quick brown fox jumps over the lazy dog");

		if (pangram)
			Console.WriteLine("Given string is a pangram");
		else
			Console.WriteLine("Given string is not a pangram");
	}
}

File: is_pangram/is_pangram.py
def is_pangram(string):

	pangram = [False] * 26

	for i in range(len(string)):
		if string[i] >= 'A' and string[i] <= 'Z':
			pangram[ord(string[i]) - ord('A')] = True
		if string[i] >= 'a' and string[i] <= 'z':
			pangram[ord(string[i]) - ord('a')] = True

	for i in pangram:
		if not i:
			return False

	return True

def main():
	pangram = is_pangram('The quick brown fox jumps over the lazy dog')

	if pangram:
		print('Given string is a pangram')
	else:
		print('Given string is not a pangram')

if __name__ == '__main__':
	main()
